test

```diff
@@ -53,6 +53,7 @@ if [ -n "${csplit}" ] && [ -n "${head}" ]; then
     touch tmpdocs/files-dir/{config.ign,ca.pem,example.conf,example.service,file,file-epilogue,local-file3}
     echo "ssh-rsa AAAA" > tmpdocs/files-dir/id_rsa.pub
     echo "ssh-ed25519 AAAA" > tmpdocs/files-dir/id_ed25519.pub
+    echo '{"ignition": {"version": "3.6.0-experimental"}}' > tmpdocs/files-dir/ignition.ig
```

Comment:
Let's use a stable one here so that it does not break when we stabilise things.

```suggestion
    echo '{"ignition": {"version": "3.5.0"}}' > tmpdocs/files-dir/ignition.ign
```